iotdb/src/main/java/org/apache/iotdb/db/metadata/MTree.java

```diff
@@ -212,6 +212,34 @@ public void setStorageGroup(String path) throws PathErrorException {
     setDataFileName(path, cur);
   }
 
+  /**
+   * check whether the input path is storage group or not
+   * @param path input path
+   * @return if it is storage group, return true. Else return false
+   */
+  public boolean checkStorageGroup(String path) {
+    String[] nodeNames = path.split(DOUB_SEPARATOR);
+    MNode cur = root;
+    if (nodeNames.length <= 1 || !nodeNames[0].equals(root.getName())) {
```

Comment:
root.getName().equals(!nodeNames[0]) better? 
since root.getName() is not null

Reply:
I don't see the difference?
This is same as the implementation of setStorageGroup().

Reply:
I just want to advice that if you use A.equals(B) statement, make sure A is not null. since root.getName() is not null, change this order maybe better?

Reply:
I see, but only when "nodeNames.length > 1" the memtioned statement will be executed, so nodeNames[0] will not be null as well. The order is same as many implementations in MTree, so it should be fine.